tasks/base.py

```diff
@@ -7,8 +7,9 @@ class BaseTest(object):
     def download_mspec(self):
         if not os.path.isdir("../mspec"):
             run("cd .. && git clone --depth=100 --quiet https://github.com/rubyspec/mspec")
+            run("cd ../mespec && git checkout v1.6.0")
```

Comment:
@lucasallan I think @timfel is referring to `cd ../mespec`, which should probably be `cd ../mspec`.